editor/debugger/script_editor_debugger.cpp

```diff
@@ -854,8 +857,9 @@ void ScriptEditorDebugger::_notification(int p_what) {
 			error_tree->connect(SceneStringName(item_selected), callable_mp(this, &ScriptEditorDebugger::_error_selected));
 			error_tree->connect("item_activated", callable_mp(this, &ScriptEditorDebugger::_error_activated));
 			breakpoints_tree->connect("item_activated", callable_mp(this, &ScriptEditorDebugger::_breakpoint_tree_clicked));
-			[[fallthrough]];
-		}
+			connect("started", callable_mp(expression_evaluator, &EditorExpressionEvaluator::on_start));
```

Comment:
Is removing `fallthrough` correct? It was there to apply the style via `NOTIFICATION_THEME_CHANGED`, so unless something changed and `NOTIFICATION_THEME_CHANGED` is always sent on enter tree we still need it (or we can move the code to a `_udpate_theme` function and than call it both from there and from the theme changed notification)

Reply:
> unless something changed and NOTIFICATION_THEME_CHANGED is always sent on enter tree

Yes, it's been like that for a long time.
See also #89746